packages/@angular/cli/blueprints/ng/files/__path__/app/app.component.ts

```diff
@@ -3,9 +3,38 @@ import { Component } from '@angular/core';
 @Component({
   selector: '<%= prefix %>-root',<% if (inlineTemplate) { %>
   template: `
-    <h1>
-      {{title}}
-    </h1><% if (routing) { %>
+    <!--The content below can be removed with the new code.-->
+    <div style="text-align:center">
+      <h1>
+        Welcome to {{title}}!!
+      </h1>
+      <img width="300" src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4N
```

Comment:
@clydin notice below that I referenced your PR. I had to do a similar fix in my PR so that the tests will pass, so I've just fixed the `Welcome to {{title}}!` so that it would work and wouldn't have merge-conflicts with your PR if mine happens to go in master before yours.